concepts/05 Widgets/Form/01 Getting Started with Form/10 Create a Form.md

```diff
@@ -0,0 +1,145 @@
+---
+##### jQuery
+
+[Add DevExtreme to your jQuery application](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/00%20Add%20DevExtreme%20to%20a%20jQuery%20Application.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/') and use the following code to create a **Form**:
+    
+    <!-- tab: index.js -->
+    $(function() {
+        $("#form").dxForm({
+            // Configuration goes here
+        });
+    });
+
+    <!-- tab: index.html -->
+    <html>
+        <head>
+            <!-- ... -->
+            <script type="text/javascript" src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+
+            <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_2/css/dx.common.css">
+            <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_2/css/dx.light.css">
+            <link rel="stylesheet" href="index.css">
+
+            <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_2/js/dx.all.js"></script>
+            <script type="text/javascript" src="index.js"></script>
+        </head>
+        <body class="dx-viewport">
+            <div id="form"></div>
+        </body>
+    </html>
+
+    <!-- tab: index.css -->
+    #form {
+        height: 400px;
+    }
+
+##### Angular
+
+[Add DevExtreme to your Angular application](/concepts/40%20Angular%20Components/10%20Getting%20Started/03%20Add%20DevExtreme%20to%20an%20Angular%20CLI%20Application '/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/') and use the following code to create a **Form**:
+
+    <!-- tab: app.component.html -->
+    <dx-form id="form"
+        <!-- Configuration goes here -->
```

Comment:
May be <!-- Configuration goes here --> should be placed after '>' tag?

Reply:
Sure. Will fix it.